README.md

```diff
@@ -298,6 +298,72 @@ SCANNERS_LIST="grype" BACKEND_HOST=localhost:9999 BACKEND_DISABLE_TLS=true kubec
 ANALYZER_LIST="syft" kubeclarity-cli analyze nginx:latest -o nginx.sbom --merge-sbom inputsbom.xml
 ```
 
+### Remote scanner servers
```

Comment:
Maybe we should put this information in a separate readme and point to it from the main one? I think that this is relevant for a small percent of the users. Maybe better to keep the readme only with the essential instructions (installation and cli instructions)?